mycore.org/static/mycore_eclipse_config_script/mycore_eclipse_config.ahk

```diff
@@ -193,83 +200,130 @@ MyCoReJavaCodeStyle() {
 
 ; XML-Code Formatierung
 XMLCode() {
-	; Window -> Preferences -> XML -> XML Files -> Editor
-	Send, ^3
-	Send, Editor XML XML Files
-	Send, {Enter}
-	; Standardeinstellungen setzen
-	Send, !a
-	Send, {Tab}
-	; Line width 120
-	Send, !w
-	Send, ^a
-	Send, 120
-	; Indent using spaces
-	Send, !n
-	; Indentation size 2
-	Send, !d
-	Send, 2
-	; Format Comments abschalten
-	Send, !f
-	Send, -
-	;Apply Button klicken
-	Send, !a
-	Send, {Space}
-	; Fenster schließen
-	Send, {Esc}
-}
+	global EclipseVersion
+	if (EclipseVersion >= "2024-12") {
+	; Window -> Preferences -> XML -> Formatting
+		Send, ^3
+		Send, XML Wild Formatting
+		Send, {Enter}
+		; Standardeinstellungen setzen
+		Send, !a
+		; Shift Tab. Ein Tab zur�ck
+		Send, +{Tab}
+		Send, {Enter}
+		; Line width 120
+		Send, !e
+		Send, +{Tab}, +{Tab}
+		Send, ^a
+		Send, 120
+		; Indent using spaces
 
-; HTML-Code Formatierung
-HTMLCode() {
-	global ConfigEclipseVersion
-	if (ConfigEclipseVersion != "2022-12") {
-		; Window -> Preferences -> Web -> HTML Files -> Editor
+	} else {
+		; Window -> Preferences -> XML -> XML Files -> Editor
 		Send, ^3
-		Send, Editor Web HTML Files
+		Send, Editor XML XML Files
 		Send, {Enter}
 		; Standardeinstellungen setzen
 		Send, !d
+		Send, {Tab}
 		; Line width 120
 		Send, !w
 		Send, ^a
 		Send, 120
 		; Indent using spaces
 		Send, !n
 		; Indentation size 2
-		Send, !t
+		Send, !d
 		Send, 2
-		; Apply Button klicken
+		; Format Comments abschalten
+		Send, !f
+		Send, -
+		;Apply Button klicken
 		Send, !a
 		Send, {Space}
 		; Fenster schließen
 		Send, {Esc}
 	}
-
 }
 
-; CSS-Code Formatierung
-CSSCode() {
-		; Window -> Preferences -> Web -> CSS Files -> Editor
+; HTML-Code Formatierung
+HTMLCode() {
+	global EclipseVersion
+	if (ConfigEclipseVersion != "2022-12") {
```

Comment:
hier mit if (EclipseVersion < "2024-12") nochmal deutlich machen, dass das nicht mehr gültig ist.
Ggf. Nachricht "Die Konfiguration für HTML-Editioren wurde in Eclipse 2024.12 entfernt".